### PR TITLE
WIP refactor: simplify api

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -16,11 +14,17 @@ export async function query(graphqlQuery, variables) {
...
   return json
 }
-
-export async function tasks(variables) {
-  return query(Tasks, variables)
-}
-
-export async function createTask(variables) {
-  return query(CreateTask, variables)
+export async function mutate(graphqlQuery, variables) {
```
Comment (riseshia) [inline src/api/index.js:17]: Is this required? 🤔 This looks as same as `query`

Comment (marocchino) [reply src/api/index.js:17]: for now, I will generate uuid in it later this pr.

Comment (riseshia) [reply src/api/index.js:17]: 🙆‍♂️

**Review by riseshia — APPROVED** [review]

**Comment by marocchino** [comment]
T-T WIP means It is not over yet.

**Comment by riseshia** [comment]
Ooops 🙈

**Pull request merged**